### DISCUSSION TITLE
How to set "place" plugin ignore specific window?

### DISCUSSION THREAD

**Discussion by shootheart** [WayfireWM/wayfire#2501] [Q&A]
I have set "random" on place mode, and I want to disable it on some windows (such as popup diag).

**Comment by ammen99** [comment]
Dialogs like for example file chooser dialog, 'are you sure you want to exit', etc. should be automatically recognized as such by wayfire and they shouldn't be subject to the place plugin at all. Which app(s) were causing these issues?
